pkg/controller/lifecycle/actuator.go

```diff
@@ -134,8 +115,7 @@ func (a *actuator) ForceDelete(ctx context.Context, _ logr.Logger, ex *extension
 		return errors.New("cluster.shoot is not yet populated")
 	}
 
-	// TODO(plkokanov): remove this after a couple releases
-	return managedresources.DeleteForShoot(ctx, a.client, namespace, constants.ManagedResourceName)
+	return nil
```

Comment:
This function currently doesn't do anything. You can remove the rest of the code and simply return `nil`.